infinity/formats/itm_v1.py

```diff
@@ -485,7 +628,7 @@ class ITM_V10_Format (Format):
             { 'key': 'opcode_number',
               'type': 'WORD',
               'off': 0x0000,
-              'enum': 'effects',
+              'enum': 'effects', # FIXME: this ids file does not exist
               'label': 'Opcode number'},
```

Comment:
It does. What I meant is to ship prepared files as external resource with iesh and load them up if the ids is not available in the game data (i.e., it's not gemrb).